parquet-tools/src/main/java/org/apache/parquet/tools/util/MetadataUtils.java

```diff
@@ -149,6 +146,13 @@ private static void showDetails(PrettyPrintWriter out, ColumnChunkMetaData meta,
     out.format(" FPO:%d", foff);
     out.format(" SZ:%d/%d/%.2f", tsize, usize, ratio);
     out.format(" VC:%d", count);
+
+    if (!meta.getStatistics().isEmpty()) {
+      out.format(" MIN:%s", meta.getStatistics().genericGetMin().toString());
+      out.format(" MAX:%s", meta.getStatistics().genericGetMax().toString());
```

Comment:
@majetideepak You are right. I updated the code. Now It should work as you have suggested.